### ISSUE TITLE
The loss calculation of PSPNet

### ISSUE THREAD

**Issue by Rhyssiyan** [ycszen/TorchSeg#113]
For the implementation of PSPNet, e.g. ade.pspnet.R101_v1c folder, CrossEntropyLoss is adopted as the criterion for the PSPNet(train.py L49). However, the prediction, which is then fed into the criterion, has been transformed with log_softmax(network.py L50). If I don't miss something, should the correct criterion used here be NLLLoss?
